src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../../components/Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleTable } from '../../components/PeopleTable/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
```

Comment:
The initial state for 'people' is set to 'null'. Consider initializing it as an empty array '[]' to avoid potential issues when checking 'people?.length'.